bugbug/model.py

```diff
@@ -30,7 +30,29 @@ def __init__(self, lemmatization=False):
     def get_feature_names(self):
         return []
 
-    def train(self):
+    def get_important_features(self, cutoff, shap_values):
+        # Calculate the values that represent the fraction of the model output variability attributable
+        # to each feature across the whole dataset.
+        shap_sums = shap_values.sum(0)
+        abs_shap_sums = np.abs(shap_values).sum(0)
+        rel_shap_sums = abs_shap_sums / abs_shap_sums.sum()
+
+        cut_off_value = cutoff * np.amax(rel_shap_sums)
+
+        # Get indices of features that pass the cut off value
+        top_feature_indices = np.where(rel_shap_sums >= cut_off_value)[0]
+        # Get the importance values of the top features from their indices
+        top_features = np.take(rel_shap_sums, top_feature_indices)
+        # Get the shap_sums of the top features
+        top_shap_sums = np.take(shap_sums, top_feature_indices)
+        # Stack the importance, indices and shap_sums in a 2D array
+        top_features = np.column_stack((top_features, top_feature_indices, top_shap_sums))
```

Comment:
Since we're not really using the shap_sums, can we have an array of True/False (1/0) to tell whether the feature is positive or negative?